blaze-server/src/main/scala/org/http4s/server/blaze/BlazeServerBuilder.scala

```diff
@@ -105,8 +106,9 @@ class BlazeServerBuilder[F[_]](
     httpApp: HttpApp[F],
     serviceErrorHandler: ServiceErrorHandler[F],
     banner: immutable.Seq[String],
-    val channelOptions: ChannelOptions
-)(implicit protected val F: ConcurrentEffect[F], timer: Timer[F])
+    val channelOptions: ChannelOptions,
+    D: Dispatcher[F]
```

Comment:
How about a `withDispatcher`, too?  I think we're doing that on other backends that take one.

Reply:
Missed it!

Reply:
Has a naming convention been settled on for `Dispatcher`? I don't really like `D` because, though it looks like one, it's not a typeclass instance, so I'm inclined to change everything to `dispatcher`

Reply:
I like `dispatcher`.